### DISCUSSION TITLE
Increase the usage of tooltips

### DISCUSSION THREAD

**Discussion by hiimjako** [mia-platform/community#148] [Ideas]
Hi,

I was wondering if it could be possible to increase the usage of tooltips on names hover to improve the UI readability.
Some examples could be:
- In console homepage if the project has a long name, it will be truncated, forcing you to open the project tab to see it complete,
- Inside the design view of a project, if the branch name is too long there is no way to expand it, and you are forced to open it on GitLab;
I think that this need may also arise for other cases not yet encountered.

A fast solution could be the usage of the `title` attribute in the HTML tags. 

Thanks

**Comment by fredmaggiowski** [comment]
Hi, thanks for your feedback, we've already planned for some readability improvements. @albertotessarotto, let's make sure these two are tracked as well (I actually think they are already).

**Comment by albertotessarotto** [comment]
Thanks for the issue.
Both for projects card and for branch name, a related activity is already in our backlog. We are going to plan it soon and let you know
